pdns/dnsdistdist/m4/systemd.m4

```diff
@@ -0,0 +1,148 @@
+# systemd.m4 - Macros to check for and enable systemd          -*- Autoconf -*-
+#
+# Copyright (C) 2014 Luis R. Rodriguez <mcgrof@suse.com>
+#
```

Comment:
Can we note the origin (including revision) of this file (or whatever we replace it with to fix @rgacogne's report)?